airflow/providers/microsoft/azure/secrets/key_vault.py

```diff
@@ -168,7 +168,11 @@ def build_path(path_prefix: str, secret_id: str, sep: str = '-') -> str:
         :param secret_id: Name of the secret
         :param sep: Separator used to concatenate path_prefix and secret_id
         """
-        path = f'{path_prefix}{sep}{secret_id}'
+        # when no prefix is given by default it adds sep for the key
```

Comment:
I find this comment a big confusing/misleading. When no prefix is given then we don't use sep at all (which I agree is the right thing):
```suggestion
        # When an empty prefix is given, do not add a separator to the secret name
```